include/ensmallen_bits/callbacks/early_stop_at_min_loss.hpp

```diff
@@ -28,11 +33,34 @@ class EarlyStopAtMinLoss
    * @param patienceIn The number of epochs to wait after the minimum loss has
    *    been reached or no improvement has been made (Default: 10).
    */
-  EarlyStopAtMinLoss(const size_t patienceIn = 10) :
-      patience(patienceIn),
+  EarlyStopAtMinLoss<MatType>(
+      const size_t patienceIn = 10)
+    : callbackUsed(false), 
```

Comment:
```suggestion
  EarlyStopAtMinLoss<MatType>(const size_t patienceIn = 10) :
      callbackUsed(false), 
```

This should fit in a single line.